app/apollo/schema/subscription.js

```diff
@@ -112,12 +113,12 @@ const subscriptionSchema = gql`
      """
      Adds a subscription
      """
-     addSubscription(orgId: String! @sv, name: String! @sv, groups: [String!]! @sv, channelUuid: String! @sv, versionUuid: String! @sv): AddChannelSubscriptionReply!
+     addSubscription(orgId: String! @sv, name: String! @sv, groups: [String!] @sv, channelUuid: String! @sv, versionUuid: String! @sv, clusterId: String): AddChannelSubscriptionReply!
```

Comment:
does the string validation ( @sv ) function need to be applied to clusterId here?   same with line 121 for editSubscription